nautobot/core/tests/runner.py

```diff
@@ -55,31 +57,79 @@ def setup_test_environment(self, **kwargs):
             setup_nautobot_job_logging(None, None, app.conf)
 
     def setup_databases(self, **kwargs):
-        result = super().setup_databases(**kwargs)
-
-        if settings.TEST_USE_FACTORIES and result:
-            command = ["generate_test_data", "--flush", "--no-input"]
-            if settings.TEST_FACTORY_SEED is not None:
-                command += ["--seed", settings.TEST_FACTORY_SEED]
-            if self.cache_test_fixtures:
-                command += ["--cache-test-fixtures"]
-            for connection in result:
-                db_name = connection[0].alias
-                print(f'Pre-populating test database "{db_name}" with factory data...')
-                db_command = [*command, "--database", db_name]
-                call_command(*db_command)
-
-        return result
+        time_keeper = self.time_keeper
```

Comment:
Does vendoring django's methods here mean more work for us when we upgrade to 4.0? We should probably keep a note somewhere.

Reply:
It definitely does make for something we need to maintain, though I did a quick compare in github between 3.2.x and 4.2.x implementations and it looks like the only significant change is https://github.com/django/django/commit/3089018e951dcca568574c0e0ebf9d8aab112389#diff-03e86b809c687971a91f0f1aeee4e5731251c2f05956f8c46d7a08e63655b67fL159-L186